GsaGH/Parameters/5_Results/4_Subsets/NodeResultsUtility.cs

```diff
@@ -121,7 +121,7 @@ internal static IQuantity Envelope<T>(this IEnumerable<T> subset, EnvelopeMethod
     internal static Ratio Envelope(this IEnumerable<Ratio?> subset, EnvelopeMethod envelopeType) {
       var val = new Ratio(0, RatioUnit.DecimalFraction);
       for (int i = 0; i < subset.Count(); i++) {
-        if (subset.ElementAt(i) != null) {
+        if (subset.ElementAt(i).HasValue) {
```

Comment:
we don't support Nulls on Ratio?

Reply:
API return either true vale, NaN or Infinity.